relay-server/src/services/processor/replay.rs

```diff
@@ -118,29 +109,32 @@ pub fn process(
     Ok(())
 }
 
+#[derive(Debug)]
+struct ReplayState<'a> {
+    pub config: &'a ProjectConfig,
+    pub global_config: &'a GlobalConfig,
+    pub geoip_lookup: Option<&'a GeoIpLookup>,
+    pub event_id: Option<EventId>,
+    pub project_id: Option<ProjectId>,
+    pub organization_id: Option<u64>,
+    pub client_addr: Option<IpAddr>,
+    pub user_agent: RawUserAgentInfo<String>,
```

Comment:
What does `pub` do on a field of a private struct?

Reply:
Just 'documentation' that these fields can be freely accessed as opposed to stuff which should only be exposed via methods.